### DISCUSSION TITLE
level of access to users

### DISCUSSION THREAD

**Discussion by felipe-dr** [decaporg/decap-cms#6933] [General]
Good night people,

I would like to confirm that CMS Decap nowadays does not have a way to define the access level (Roles) for each user, for example:

**admin -->** create, publish, read, update, delete **any** collection
**author -->** just create, but don't publish to master, read **some** collections

I searched and found some old asks on the forum, but I would like to confirm, please.

Thanks in advance.

Felipe D. R

**Comment by martinjagodic** [comment]
Nope, this is the oldest open issue #2
